conans/client/cmake.py

```diff
@@ -40,7 +40,8 @@ def _generator(self):
                         '10': '10 2010',
                         '11': '11 2012',
                         '12': '12 2013',
-                        '14': '14 2015'}
+                        '14': '14 2015',
+                        '15': '15 2017'}
```

Comment:
I see only one issue with that if a package was released for VS2015 and their author not active. IDK how to use this package with VS2017